libs/ui/src/lib/notice/notice.component.stories.ts

```diff
@@ -0,0 +1,281 @@
+import { Story, Meta } from '@storybook/angular';
+import notes from './notice.component.md';
+import { NoticeComponent } from './notice.component';
+
+export default {
+  title: 'Angular/Notice',
+  parameters: { notes, layout: 'padded' },
+  argTypes: {
+    severity: {
+      name: 'Severity style',
+      options: ['info', 'success', 'error'],
+      control: { type: 'select' },
+    },
+  },
+  args: {
+    severity: 'info',
+  },
+} as Meta<NoticeComponent>;
+
+const Template: Story<NoticeComponent> = (args: NoticeComponent) => ({
+  props: args,
+  template: `
+    <cvi-ng-notice [iconName]="iconName" [severity]="severity">
+      <cvi-ng-track
+        layout="flex"
+        horizontalAlignment="justify"
+        verticalAlignment="center"
+      >
+        any content content
+      </cvi-ng-track>
+    </cvi-ng-notice>
+  `,
+});
+
+export const Default = Template.bind({});
+
+const TemplateWithError: Story<NoticeComponent> = (args: NoticeComponent) => ({
+  props: args,
+  template: `
+    <cvi-ng-notice [iconName]="iconName" [severity]="'error'">
+      <cvi-ng-track
```

Comment:
Should we incorporate more of the nested markup directly inside of the component? The current way to implement a typical use case for this component looks to complex – there's simply too much stuff to copy from a code example. 

Looking at the design in Figma, these elements always appear in any variation of the component design: 

- Title (could be `title` prop)
- Date in the left part (could be `date` prop)
- Time in the left part (could be `time` prop)
- Button to the right - notably it changes its `appearance` according to notice severity, this would be a good candidate for encapsulation

The remaining slots could be: 

- `subtitle` slot – for an org name and an icon under the title
- `aside` slot – for another date, price etc on the right, but excluding the button